packages/backend/src/core/activity/processors/RpcProcessor.ts

```diff
@@ -24,7 +24,7 @@ export function createRpcProcessor(
   const batchSize = getBatchSizeFromCallsPerMinute(callsPerMinute)
   const url = transactionApi.url
   assert(url, 'Url for rpc client must be defined')
-  const provider = new providers.JsonRpcProvider({
+  const provider = new providers.StaticJsonRpcProvider({
```

Comment:
Can we do that for activity v2 also?

Reply:
This is actually the v2, updated v1 as well